apps/console/src/features/claims/constants/claim-management-constants.ts

```diff
@@ -119,7 +119,12 @@ export class ClaimManagementConstants {
         SCIMConfigs.oidc
     ];
 
-    public static readonly SCIM_MAPPING: string[] = Object.values(SCIMConfigs.scim);
+    public static readonly SCIM_MAPPING: string[] = [
```

Comment:
Any reason for duplicating this?
`SCIMConfigs.scim` seems to have the values.
Can't we use `Object.values(SCIMConfigs.scim)`?

Reply:
`SCIM config` file is used to map scim attributes.
`claim-management-constants` file is used for rendering.
Values of these two files can differ in different usecases. Better to have in two files.

Reply:
Fixed with e77c40b16a7ebe4406558279bd52cfd5ac39181e